src/jmeter.ts

```diff
@@ -0,0 +1,214 @@
+    },
+    {
+      name: ["-E", "--proxyScheme"],
+      description: "Set a proxy scheme to use for the proxy server",
+      args: {
+        isOptional: false,
+        name: "proxyScheme",
+        description: "Set a proxy scheme to use for the proxy server",
+      },
+    },
+    {
+      name: ["-H", "--proxyHost"],
+      description: "Set a proxy server for JMeter to use",     
+    },
+    {
+      name: ["-P", "--proxyPort"],
+      description: "Set proxy server port for JMeter to use",      
+    },
+    {
+      name: ["-N", "--nonProxyHosts"],
+      description: "Set nonproxy host list (e.g. *.apache.org|localhost)",
+      args: {
+        isOptional: false,
+        name: "nonProxyHosts",
+        description: "Set nonproxy host list (e.g. *.apache.org|localhost)",
+      },
+    },
+    {
+      name: ["-u", "--username"],
+      description: "Set username for proxy server that JMeter is to use",
+      args: {
+        isOptional: false,
+        name: "username",
+        description: "Set username for proxy server that JMeter is to use",
+      },
+    },
+    {
+      name: ["-a", "--password"],
+      description: "Set password for proxy server that JMeter is to use",
+      args: {
+        isOptional: false,
+        name: "password",
+        description: "Set password for proxy server that JMeter is to use",
+      },
+    },
+    {
+      name: ["-J", "--jmeterproperty"],
+      description: "Define additional JMeter properties <argument>=<value>",
+      args: {
+        isOptional: false,
+        name: "jmeterproperty",
+        description: "Define additional JMeter properties <argument>=<value>",
+      },
+    },
+    {
+      name: ["-G", "--globalproperty"],
+      description: "Define Global properties (sent to servers) e.g. -Gport=123 or -Gglobal.properties",
+      args: {
+        isOptional: false,
+        name: "globalproperty",
+        description: "Define Global properties (sent to servers) e.g. -Gport=123 or -Gglobal.properties",
+      },
+    },
+    {
+      name: ["-D", "--systemproperty"],
+      description: "Define additional system properties <argument>=<value>",
+      args: {
+        isOptional: false,
+        name: "systemproperty",
+        description: "Define additional system properties <argument>=<value>",
+      },
+    },
+    {
+      name: ["-S", "--systemPropertyFile"],
+      description: "additional system property file(s)",
+      args: {
+        isOptional: false,
+        name: "systemPropertyFile",
+        description: "additional system property file(s)",
+      },
+    },
+    {
+      name: ["-f", "--forceDeleteResultFile"],
+      description: "force delete existing results files and web report folder",      
+    },
+    {
+      name: ["-L", "--loglevel"],
+      description: "[category=]level e.g. jorphan=INFO, jmeter.util=DEBUG or com.example.foo=WARN",
+      args: {
+        isOptional: false,
+        name: "loglevel",
```

Comment:
This should use the [`keyValue`](https://fig.io/docs/reference/generator/prebuilt-generators#keyvalueoptions) generator
```suggestion
        name: "loglevel",
        generators: keyValue({
	        values: ["INFO", "WARN", "DEBUG"],
	      }),
```